pyscal/utils/interpolation.py

```diff
@@ -398,7 +398,7 @@ def weighted_value(a, b):
 
 
 def interpolator(
-    tableobject, wo_low, wo_high, parameter, sat="sw", kr1="krw", kr2="krow", pc="pc"
+    tableobject, wo_low, wo_high, parameter, sat="SW", kr1="KRW", kr2="KROW", pc="pc"
```

Comment:
Here "pc" is still lower case, will this work? If not, why was this not picked up by the tests?

Reply:
Fixed (but this is a deprecated and untested function that is deleted in #228 )